crypto/pkcs11key/key_test.go

```diff
@@ -5,21 +5,32 @@ package pkcs11key
 import (
 	"bytes"
 	"crypto"
+	"crypto/ecdsa"
 	"crypto/rand"
-	"testing"
-
+	"crypto/rsa"
 	"github.com/miekg/pkcs11"
+	"testing"
 )
 
 type mockCtx struct{}
 
-const privateKeyHandle = pkcs11.ObjectHandle(23)
+const rsaPrivateKeyHandle = pkcs11.ObjectHandle(23)
+const ecPrivateKeyHandle = pkcs11.ObjectHandle(32)
+const ecPublicKeyHandle = pkcs11.ObjectHandle(33)
+
+// EC private key with no matching public key
+const ecPrivNoPubHandle = pkcs11.ObjectHandle(34)
+
+// EC private and public key with invalid EC point
+const ecInvEcPointPrivHandle = pkcs11.ObjectHandle(35)
+const ecInvEcPointPubHandle = pkcs11.ObjectHandle(36)
 const sessionHandle = pkcs11.SessionHandle(17)
 
 var slots = []uint{7, 8, 9}
 var tokenInfo = pkcs11.TokenInfo{
 	Label: "token label",
 }
+var currentSearch []*pkcs11.Attribute
```

Comment:
This should be an attribute of `mockCtx` rather than a global variable. That way multiple tests can run in parallel.